client/js/components/assessmenttable/DpNewStatement.vue

```diff
@@ -230,8 +248,9 @@ export default {
     },
 
     setPhaseValue (value) {
+      console.log('setPhaseValue', value)
```

Comment:
please remove this

Reply:
thanks. https://github.com/demos-europe/demosplan-core/pull/3916/commits/9a6d41541440e53072e864d94291c135b606a9d7